scripts/import-digiroad-data.sh

```diff
@@ -1,12 +1,32 @@
-#!/bin/sh
+#!/bin/bash
 
 set -e
+available_provinces="'uusimaa' and 'varisinais-suomi'"
 
-# URL to a zip file containing .shp files
-DATA_URL=https://aineistot.vayla.fi/digiroad/latest/Maakuntajako_DIGIROAD_R_EUREF-FIN/UUSIMAA.zip
+if [ $# == 0 ]
+then
+    echo "No province argument supplied."    
+    echo "Available provinces are ${available_provinces}."
+    exit 0
+fi
 
+declare -A package_name
+package_name["varsinais-suomi"]="VARSINAIS-SUOMI.zip"
+package_name["uusimaa"]="UUSIMAA.zip"
+
+if [ ${package_name[$1]+_} ] 
+then 
+    echo "Importing Digiroad data for province $1."; 
+else 
+    echo "Province $1 not found, available provinces are ${available_provinces}."; 
+    exit 0
+fi
+
+echo ${package_name[$1]}
+
+DATA_URL="https://ava.vaylapilvi.fi/ava/Tiest%C3%B6tiedot/Digiroad/Digiroad-irrotusaineistot/latest/Maakuntajako_DIGIROAD_R_EUREF-FIN/${package_name[$1]}"
 # Directory where the data will be downloaded and extracted
-DATA_DIR=/tmp/digiroad
+DATA_DIR=/home/juuso/repos/geo-search/scripts/tmp/digiroad
```

Comment:
I think this should be reverted back to use the regular `/tmp/digiroad`-folder

Reply:
That has been done.